components/src/molecules/Tabs.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import { Tabs as TabComponent } from './Tabs'
+import type { Meta, StoryObj } from '@storybook/react'
+import { useArgs } from '@storybook/preview-api';
```

Comment:
nit: we are very particular about import order, the pattern goes:
- package imports
- top level directory imports (like `opentrons/shared-data` or `opentrons/components`)
- imports from the farthest path to the shortest so `../../` should be above `./` for example
- types (type imports should also follow the above pattern)

```suggestion
import * as React from 'react'
import { useArgs } from '@storybook/preview-api';
import { Tabs as TabComponent } from './Tabs'
import type { Meta, StoryObj } from '@storybook/react'
```

Reply:
thanks jethary, i forgot to explain this to shiyao